README.md

```diff
@@ -1,15 +1,42 @@
 
-[![Matrix Space](https://img.shields.io/static/v1?label=Space&message=%23Wgpu&color=blue&logo=matrix)](https://matrix.to/#/#Wgpu:matrix.org)
 [![Dev Matrix](https://img.shields.io/static/v1?label=devs&message=%23wgpu&color=blueviolet&logo=matrix)](https://matrix.to/#/#wgpu:matrix.org)
 [![User Matrix](https://img.shields.io/static/v1?label=users&message=%23wgpu-users&color=blueviolet&logo=matrix)](https://matrix.to/#/#wgpu-users:matrix.org)
 [![Build Status](https://img.shields.io/github/actions/workflow/status/gfx-rs/wgpu/ci.yml?branch=trunk&logo=github&label=CI)](https://github.com/gfx-rs/wgpu/actions)
 [![codecov.io](https://img.shields.io/codecov/c/github/gfx-rs/wgpu?logo=codecov&logoColor=fff&label=codecov&token=84qJTesmeS)](https://codecov.io/gh/gfx-rs/wgpu)
 
 `wgpu` is a cross-platform, safe, pure-rust graphics API. It runs natively on Vulkan, Metal, D3D12, and OpenGL; and on top of WebGL2 and WebGPU on wasm.
 
-The API is based on the [WebGPU standard][webgpu]. It serves as the core of the WebGPU integration in Firefox, Servo, and Deno.
+The API is based on the [WebGPU standard][webgpu], but is a fully native rust library. It serves as the core of the WebGPU integration in Firefox, Servo, and Deno.
+
+## Getting Started
+
+See our examples online at <https://wgpu.rs/examples/>. You can see the Rust sources at [examples](examples) and run them directly with `cargo run --bin wgpu-examples <example>`.
+
+### Learning `wgpu`
+
+If you are new to wgpu and graphics programming, we recommend starting with https://sotrh.github.io/learn-wgpu/. 
+
+Additionally, <https://webgpufundamentals.org/> is a tutorial for WebGPU which is very similar to our API, minus differences between Rust and Javascript.
+
+### Wiki
+
+We have a [wiki](https://github.com/gfx-rs/wgpu/wiki) which has information on useful architecture patterns, debugging tips, and more getting started information. 
+
+### Need Help? Want to Contribute? 
+
+The wgpu community uses `Matrix` to discuss.
+
+- [![Wgpu Matrix](https://img.shields.io/static/v1?label=wgpu-devs&message=%23wgpu&color=blueviolet&logo=matrix)](https://matrix.to/#/#wgpu:matrix.org) - discussion of wgpu's development.
+- [![User Matrix](https://img.shields.io/static/v1?label=wgpu-users&message=%23wgpu-users&color=blueviolet&logo=matrix)](https://matrix.to/#/#wgpu-users:matrix.org) - discussion of using the library and the surrounding ecosystem.
+- [![Random Matrix](https://img.shields.io/static/v1?label=random&message=%23wgpu-random&color=blueviolet&logo=matrix)](https://matrix.to/#/#wgpu-random:matrix.org) - discussion of everything else.
+
+### Other Languages
+
+To use wgpu in C or dozens of other languages, look at [wgpu-native](https://github.com/gfx-rs/wgpu-native). These are C bindings to wgpu and has an up to date list of libraries bringing support to other languages. 
+
+<https://eliemichel.github.io/LearnWebGPU/> is a good resource for learning how to use wgpu-native from C.
```

Comment:
thought: Maybe we could use nicer names for these resource links? I'm on the fence, because seeing the URL can be helpful, but for most folks finding these resources for the first time, a friendly name is usually less overwhelming.